### ISSUE TITLE
Question about the KNN input channel

### ISSUE THREAD

**Issue by MCLYang** [AnTao97/dgcnn.pytorch#2]
Thank you for the pytorch implementation. I was looking at the model.py and found that the input channel maybe vary.  
```
def get_graph_feature(x, k=20, idx=None, moreFeatures=True):
    batch_size = x.size(0)
    num_points = x.size(2)
    x = x.view(batch_size, -1, num_points)
    if idx is None:
        if dim9 == False:
            idx = knn(x, k=k)   # (batch_size, num_points, k)
        else:
            idx = knn(x[:, 6:], k=k)
```
I am wondering when when I should pass all features through KNN and when I should pass partial.
(According to S3DIS, the each point as 9-dim which are XYZ RGB and normalized XYZ.)

**Comment by antao97** [comment]
Actually, the original TensorFlow implementation computes KNN in the first `get_graph_feature()` like this, and I follow all steps it takes in my PyTorch version. I think why the author computes the first KNN with only normalized XYZ is because XYZ is more important than RGB color in low-level feature extraction.

**Comment by MCLYang** [comment]
@AnTao97 I believe it too. Thank you
